README.md

```diff
@@ -4,29 +4,84 @@
 
 The fastest way to install [Micro](https://micro-editor.github.io/)
 
-`curl https://getmic.ro | bash`
+```Bash
+# local install just for your user
+curl https://getmic.ro | sh
```

Comment:
I'd prefer to keep the top-line documentation referring to `bash` rather than `sh` (but keep the note below that it can be run with `sh` instead if you don't have bash) -- even though the script is tested across a few different shells and is designed to be POSIX-compliant, I think it'll be a smoother experience for folks to use `bash` if they have it installed since people can have all sorts of wacky stuff installed as `/bin/sh`. [See past discussion here](https://github.com/benweissmann/getmic.ro/pull/12).

Reply:
> I think it'll be a smoother experience for folks to use bash if they have it installed since people can have all sorts of wacky stuff installed as /bin/sh

I see your concern and I agree it's good to be cautious, however, you have nothing to fear. This "wacky stuff" is non-POSIX extensions added to scripting. As long as we don't use these extensions (and I have ensured we do not use them), then /bin/sh will execute this script just fine on every computer that exists today aside from Windows, which has neither /bin/sh nor /bin/bash.

My arguments:

* All non-tech-savvy users using a user-friendly OS distributed within the past 10 years are guaranteed to be using a POSIX-compliant interpreter at /bin/sh. However, some of them, e.x. those using Manjaro, don't have Bash installed by default. 

* If you are running such a niche OS that you have a non-compliant shell then you are accustomed to solving such trivial problems as changing the shell script interpreter to a more suitable one. It is far better to put a problem in the hands of an expert who can solve it in 5 seconds than in the hands of a beginner who will struggle all day to solve it (that is, assuming an expert will ever run into an issue, which is highly doubtful. I'm almost certain /bin/sh will give noone any issues).

* Almost all non-POSIX environments I've ever seen have been on absolutely minimal busyboxes, and even these environments still have the standard builtins like `[`, `command`, `set`, and `eval`. For compatibility with these environments, its much more important to focus on never assuming a non-builtin-command exists, which I have done. It doesn't even assume that printf exists.

* Environments quirky enough to have a non-POSIX-compliant shell scripting language are practically guaranteed to be so purpose-specific that they are running on their own unique hardware and likely don't even have curl or wget. Moreover, these are not the machines which you would see a person running everyday software on.

* I cite https://sourceforge.net/projects/judy/ as an excellent demonstration. Written in 1999 when operating systems were a lot less consistent and cross-os-compatibility was much more problematic, Judy still managed to compile just fine on tons of quirky machines such Intel's IA64 Linux server distribution. How? Because all these quicky machines had one thing in common: /bin/sh was POSIX compliant. So, Judy used /bin/sh

To quote Akianonymus from the previous conversation:

> Also, I don't see a point keeping bash as a shebang when your script is POSIX, that's just straight misleading.

The changes between versions of Bash have not been changes in POSIX, they have been changes in Bash-specific syntax. We avoid Bash in this script and only use POSIX, so we are guaranteed compatibility with all POSIX-compliant shell implementations (which is every non-Windows operating system in use today).

Reply:
I think I've said everything I have to say about this in that other thread -- I think this is definitely something that reasonable folks can disagree on! But I'm not interested in rehashing this debate; I'd like to keep the top-line README using `bash` rather than `sh`.

Reply:
Very well. I've changed it.